state-chain/traits/src/lib.rs

```diff
@@ -1189,3 +1189,26 @@ pub trait AffiliateRegistry {
 pub trait MinimumDeposit {
 	fn get(asset: Asset) -> AssetAmount;
 }
+
+#[derive(Decode, Encode, Default, Debug, TypeInfo)]
+pub struct TradingStrategyParameters {
+	pub order_update_thresholds: BoundedBTreeMap<Asset, AssetAmount, ConstU32<1000>>,
+	pub strategy_deployment_thresholds: BoundedBTreeMap<Asset, AssetAmount, ConstU32<1000>>,
+}
```

Comment:
Worth adding comments about what these mean - in particular things like the fact the deployment threshold is either/or - and not the full amount for each (ideally pointing to the code in trading-strategies where this logic is).

Reply:
These are separate storate items now. Added comments.